scripts/reporting/svn_report.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+'''
+This file creates a .csv file containing the name of each laptop and its last changed date
+'''
+import argparse
+import csv
+from datetime import datetime, timezone
+import os
+import svn.local
+import pandas as pd
+from yaml import load, dump
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+'''
+Constants -- paths for reports, default save names, SLA, columns, and sites
+TO-DO: Change SLA_DAYS to a parser arg?
+'''
+REPORTS_DIR = '/fs/storage/laptops/ncanda'
+DEFAULT_CSV = '/fs/ncanda-share/log/status_reports/sla_files/'
+SLA_DAYS = 30
+DATA_COLUMNS = ['laptop', 'date_updated', 'time_diff', 'sla', 'sla_percentage']
+SITES = ['duke', 'sri', 'ohsu', 'upmc', 'ucsd']
+CONFIG_FILE = 'svn_report_config.yml'
+
+def parse_args(arg_input=None):
+    '''
+    Set up parser arguments
+    '''
+    parser = argparse.ArgumentParser(
+        description="Create a CSV file with all laptops and dates they were last modified")
+    parser.add_argument(
+        "--file",
+        help="Path of file name to save as",
+        action="store",
+        default=DEFAULT_CSV)
+    
+    return parser.parse_args(arg_input)
+
+def load_default_sla(path=None):
+    '''
+    Loads default SLA from YAML file
+    '''
+    with open(path) as f:
+        data = load(f, Loader=Loader)
+        return data['three'], data['dead']
```

Comment:
If there's spare time, we really should refactor the config file to allow for arbitrary SLAs rather than ones that are predefined like this -- but for now, this can pass. (Here's what I mean:

```yaml
default: 30
dead: &DEAD 3000 
laptops:
  ucsd49: 3
  ucsd51: 3
  duke-merge: *DEAD

```